index.js

```diff
@@ -90,7 +90,7 @@ function transform(wanted, available) {
   wanted.forEach((browser) => {
     const name = browser.name.toLowerCase();
 
-    if (!available.has(name)) return;
+    if (!available.has(name)) throw new Error(`Browser ${name} is not available`);
```

Comment:
Can you please keep 80 chars per line?

Reply:
Sure, I wasn't sure about this since it's over 80 on other places.

Reply:
Do you want `{}` when making new lines in if-statements?

Reply:
Yes, thanks, I think it will be consistent with the rest of the code base.